gradle/gradle-mvn-publish.gradle

```diff
@@ -11,58 +11,70 @@ if (isReleaseBuild()) {
 } else {
     sonatypeRepositoryUrl = "https://oss.sonatype.org/content/repositories/snapshots/"
 }
+publishing {
+    publications {
+        maven(MavenPublication) {
+            groupId = GROUP
+            artifactId = POM_ARTIFACT_ID
+            version = VERSION_NAME
+            println components
 
-uploadArchives.repositories.mavenDeployer {
-    beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-    pom.artifactId = POM_ARTIFACT_ID
-    pom.groupId = GROUP
-    pom.version = VERSION_NAME
-
-    if (project.hasProperty('sonatypeUsername') && project.hasProperty('sonatypePassword')) {
-        repository(url: sonatypeRepositoryUrl) {
-            authentication(userName: project.sonatypeUsername, password: project.sonatypePassword)
-        }
-    }
+            afterEvaluate {
+                println components
+                from components.java
+            }
 
-    pom.project {
-        name POM_NAME
-        packaging POM_PACKAGING
-        description POM_DESCRIPTION
-        url POM_URL
+            pom {
+                name = POM_NAME
+                packaging = POM_PACKAGING
+                description = POM_DESCRIPTION
+                url = POM_URL
 
-        organization {
-            name POM_ORG_NAME
-            url POM_ORG_URL
-        }
+                organization {
+                    name = POM_ORG_NAME
+                    url = POM_ORG_URL
+                }
 
-        scm {
-            url POM_SCM_URL
-            connection POM_SCM_CONNECTION
-            developerConnection POM_SCM_DEV_CONNECTION
-        }
+                scm {
+                    url = POM_SCM_URL
+                    connection = POM_SCM_CONNECTION
+                    developerConnection = POM_SCM_DEV_CONNECTION
+                }
 
-        licenses {
-            license {
-                name POM_LICENSE_NAME
-                url POM_LICENSE_URL
-                distribution POM_LICENSE_DIST
-            }
-        }
+                licenses {
+                    license {
+                        name = POM_LICENSE_NAME
+                        url = POM_LICENSE_URL
+                        distribution = POM_LICENSE_DIST
+                    }
+                }
 
-        developers {
-            developer {
-                id POM_DEVELOPER_ID
-                name POM_DEVELOPER_NAME
-                email POM_DEVELOPER_EMAIL
+                developers {
+                    developer {
+                        id = POM_DEVELOPER_ID
+                        name = POM_DEVELOPER_NAME
+                        email = POM_DEVELOPER_EMAIL
+                    }
+                }
             }
         }
     }
+    repositories {
+        mavenLocal()
+//        maven {
```

Comment:
Delete commented code

Reply:
👍 , I've commented it out while testing pushing locally, I'll uncomment it